lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -191,7 +191,6 @@
 PuppetSyntax.exclude_paths << 'pkg/**/*'
 PuppetSyntax.exclude_paths << 'vendor/**/*'
 PuppetSyntax.exclude_paths << '.vendor/**/*'
-PuppetSyntax.exclude_paths << 'plans/**/*'
```

Comment:
That will start syntax checks for plans, right? Does that qualify for a major release?

Reply:
I initially placed a maintenance label as I wasnt entirely sure about the reach of this change but I had a short chat with the team and it sounds like it would warrant a Y release (if we treat it like expanded functionality).

Do you have any strong opinions on having a major release for this?

Reply:
If this makes pipeline fail that passed previously I think it's a major change.

Reply:
🤔

Reply:
I'm fine with what @ekohl said.